### PR TITLE
#946 Updated outdated dependencies

### LINKED ISSUE #946: dependencies are out of date
Too many deps are out of date, let's fix that, see https://www.versioneye.com/user/projects/561a9d87a193340f28000fd3

### PR DESCRIPTION
**Pull request by original-brownbear**
This resolves #946.

Versioneye shows 10 dependencies as outdated here: https://www.versioneye.com/user/projects/561a9d87a193340f28000fd3

* Updated those 10 in this PR.
* Updated Dockerfile to use Maven 3.3.3, since the newer plugins required more recent Maven than 3.2.1. 3.3.3 was the most recent I could make build on Ubuntu 14.04.
* Updated Readme accordingly, regarding the minimum Maven version.

### CODE REVIEW

**Comment by alex-palevsky** [comment]
@original-brownbear Many thanks, I will find someone to review it before we merge

**Comment by alex-palevsky** [comment]
@mkordas review this please

**Comment by mkordas** [comment]
@original-brownbear thanks, looks good!

**Comment by mkordas** [comment]
@rultor merge

**Comment by mkordas** [comment]
@original-brownbear BTW, do you plan to join our team?

**Comment by yegor256** [comment]
@rultor merge

**Comment by mkordas** [comment]
@original-brownbear can you check issue on Rultor?

**Comment by original-brownbear** [comment]
@yegor256  @mkordas I think you need to handle this one and update the Maven version in the 
Caused by: org.apache.maven.plugin.PluginIncompatibleException: The plugin nl.geodienstencentrum.maven:sass-maven-plugin:2.15 requires Maven version 3.2.5

**Comment by mkordas** [comment]
@original-brownbear let's try

**Comment by original-brownbear** [comment]
@mkordas on it

**Comment by original-brownbear** [comment]
@mkordas @yegor256 updated the Dockerfile to use Maven 3.3.3. Experienced two somewhat unforeseen issues though that need explaining:
1. I needed to raise the max heap to 2GB to make the initial build run now.
2. I needed to force this maven version to use Maven Central as mirror, the default config contained in the package ran into repeated timeouts with some Sonatype host.

=> this is the cleanest build I could come up with, using Ubuntu 14.04 and a newer Maven.
=> On the upside: Tested the build in this image via Docker run, works fine again including the Site target.

File: src/docker/Dockerfile [hunk 1]
```diff
@@ -126,6 +114,27 @@ RUN apt-get install -y nodejs npm
 RUN sudo npm install -g npm
 RUN ln -s /usr/bin/nodejs /usr/bin/node
 
+# Maven
+ENV MAVEN_VERSION 3.3.3
```
Comment (mkordas) [inline src/docker/Dockerfile]: @original-brownbear why not Maven 3.3.9?

**Comment by mkordas** [comment]
@yegor256 are you OK to update Maven in Docker image or we should rather downgrade dependency and stick to the old Maven?

File: src/docker/Dockerfile (hunk 1 above)
Comment (original-brownbear) [inline src/docker/Dockerfile]: @mkordas My oversight sorry, updating to 3.3.9.

**Comment by original-brownbear** [comment]
@mkordas pushed Maven to version 3.3.9 , tested the build locally still works fine.

**Comment by mkordas** [comment]
@original-brownbear looks OK to me, thanks! @yegor256 will make final decision

**Comment by mkordas** [comment]
@rultor merge

**Comment by mkordas** [comment]
@yegor256 can you check this pull request?

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by original-brownbear** [comment]
@yegor256 it won't work like this. If we just run the merge this way it's still ran against the old Dockerfile I'm afraid.

**Comment by yegor256** [comment]
@original-brownbear I merged it

**Comment by mkordas** [comment]
@rultor deploy

**Comment by alex-palevsky** [comment]
@rultor please deploy

**Comment by alex-palevsky** [comment]
@mkordas thank you, added **26 mins** to your acc, payment num is `56cf30bbf80fe07fd3000793`, 148 hours and 3 mins spent in total to complete... you're getting extra minutes here (c=11)... +26 to your rating, your total score is [+7176](http://www.netbout.com/b/36080?open=rating)

**Pull request merged**